linsolve/linsolve.py

```diff
@@ -250,6 +251,7 @@ def __init__(self, data, wgts={}, sparse=False, **kwargs):
         Returns:
             None
         """
+        # XXX add ability to override datatype inference
```

Comment:
I like having them in code, b/c they end up localized to certain areas where, if I'm editing, it will get my attention. However, I will see you halfway and make sure each is an issue, and that the issue is linked in the code.

Reply:
Fine with me!